src/ProjectManager.cs

```diff
@@ -117,6 +119,14 @@ public void SaveAsLua(string PAC_Name, string path, bool silentMode)
                         projectName,
                         $"\\{ProjectDescriptionSaver.MainRestrictionsFileName}");
                     techObjectManager.LoadRestriction(LuaStr);
+
+                    //Считывание таблицы рецептов
+                    errStr = string.Empty;
```

Comment:
```suggestion
                    errStr = string.Empty;
```

Где-то тут баг. Ошибки сбрасываются предыдущие.

Reply:
Но после загрузки технологических объектов и перед загрузкой ограничений мы тоже обнуляем errStr (строка 116).

Reply:
Оказывается баг повыше будет. Не работает функционал в принципе. Res = 1 возвращает только в 1 единственном случае, когда неправильный путь к файлам проекта. В остальных случаях код обнуляется или все сбрасывается. 

Если удалить main.objects.lua или main.restrictions.lua, то будет видно, что ошибки не появляются, а должны. Такая же ситуация будет и с recipe. Я переделаю, но потом придется тебе мержить изменения

Reply:
Можешь смержить изменения, в #945 переделал загрузку немножко